### PR TITLE
Replace instances of BlogTemplate for naming purposes

### PR DESCRIPTION
**Pull request by julietdaniel**
When creating an instance of the BlogTemplate, replace instances of "BlogTemplate" with "BlogTemplate.1" (for raw context) or "BlogTemplate._1" (for namespace context) so that the instance generated from the template is named as expected; the developer can customize the name when generating a new instance.

### CODE REVIEW

File: .template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "author": "Me",
```
Comment (jimmylewis) [inline .template.config/template.json]: should we have a more specific author name?

File: BlogTemplate/Pages/Index.cshtml.cs [hunk 2]
```diff
@@ -4,17 +4,16 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
-using System.IO;
-
-namespace BlogTemplate.Pages
+using BlogTemplate._1.Models;
+using System.IO;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Index.cshtml.cs]: Can you sort the usings in this file?

File: BlogTemplate.Tests/Models/BlogDataStoreTests.cs [hunk 3]
```diff
@@ -1,14 +1,14 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
-using BlogTemplate.Models;
+using BlogTemplate._1.Models;
 using Xunit;
```
Comment (jimmylewis) [inline BlogTemplate.Tests/Models/BlogDataStoreTests.cs]: sort usings

File: BlogTemplate/Pages/Drafts.cshtml.cs [hunk 4]
```diff
@@ -4,10 +4,10 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
+using BlogTemplate._1.Models;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Drafts.cshtml.cs]: sort usings

File: BlogTemplate/Pages/Edit.cshtml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 @page "{slug}"
-@using BlogTemplate
-@using BlogTemplate.Models
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Edit.cshtml]: can you reorganize these so the `@using` statements are all together and sorted?

File: BlogTemplate/Pages/Edit.cshtml.cs [hunk 6]
```diff
@@ -4,12 +4,12 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Edit.cshtml.cs]: sort usings

File: BlogTemplate/Pages/Edit.cshtml.cs [hunk 7]
```diff
@@ -4,12 +4,12 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
+using BlogTemplate._1.Models;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Edit.cshtml.cs]: sort usings

File: BlogTemplate/Pages/Index.cshtml.cs [hunk 8]
```diff
@@ -4,17 +4,16 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Index.cshtml.cs]: sort usings

File: BlogTemplate/Pages/New.cshtml [hunk 9]
```diff
@@ -1,6 +1,6 @@
 @page
-@using BlogTemplate
-@using BlogTemplate.Models
+@using BlogTemplate._1
```
Comment (jimmylewis) [inline BlogTemplate/Pages/New.cshtml]: reorganize `@using`s

File: BlogTemplate/Pages/New.cshtml.cs [hunk 10]
```diff
@@ -4,12 +4,14 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/New.cshtml.cs]: sort usings

File: BlogTemplate/Pages/Post.cshtml.cs [hunk 11]
```diff
@@ -3,13 +3,13 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using BlogTemplate.Models;
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Post.cshtml.cs]: sort usings

File: .template.config/template.json (hunk 1 above)
Comment (julietdaniel) [inline .template.config/template.json]: @lucasisaza Is this a spec issue?

File: BlogTemplate/Pages/Edit.cshtml (hunk 5 above)
Comment (julietdaniel) [inline BlogTemplate/Pages/Edit.cshtml]: I fixed this in my recent commit, but for some reason this comment isn't going away. I addressed it though

File: .template.config/template.json (hunk 1 above)
Comment (jimmylewis) [inline .template.config/template.json]: We could open an issue to track this, and take it as is for this PR.

**Comment by emanb29** [comment]
This (naturally) broke many of the unit tests. This can be confusing if you just open the project and expect the tests to run (stumped me for a few minutes, haha). Furthermore, the breakage is inconsistent (FakeFileSystem.cs and BlogDataStoreTests have been put in the _1 namespaces and use the _1 namespaces, but NewTests.cs, ExceptGeneratorTests.cs, and SlugGeneratorTests.cs have not).

Are the breaks intentional, or just a side effect of parallel PRs? If intentional, they should be documented. If not, they should be fixed.

**Pull request merged**